stdlib/either.ml

```diff
@@ -26,6 +26,14 @@ let is_right = function
 | Left _ -> false
 | Right _ -> true
 
+let get_left = function
+| Left v -> v
+| Right _ -> invalid_arg "either is Right _"
```

Comment:
The type name is not `either` but `Either.t`.

Reply:
I think the message here is by analogy to `Option.get` which returns `option is None`...

Reply:
Yes, but the shortest name of the type constructor `option` is `option`. I think that we should use `Either.t` to be consistent with `Option` and `Result` rather than referring to a non-existing type constructor.

Reply:
Good point.

Reply:
Yes, I just copy-pasted this error message from `Result` and `Option`, but those modules have their "shortened" type constructors. Fixed.